README.md

```diff
@@ -12,67 +16,100 @@ of your source files for more accurate results.
 
 ![](https://raw.githubusercontent.com/realm/SwiftLint/main/assets/screenshot.png)
 
-This project adheres to the [Contributor Covenant Code of Conduct](https://realm.io/conduct).
+This project adheres to the
+[Contributor Covenant Code of Conduct](https://realm.io/conduct).
 By participating, you are expected to uphold this code. Please report
 unacceptable behavior to [info@realm.io](mailto:info@realm.io).
 
-> Language Switch: [中文](https://github.com/realm/SwiftLint/blob/main/README_CN.md), [한국어](https://github.com/realm/SwiftLint/blob/main/README_KR.md).
+> Switch Language:
+> [中文](https://github.com/realm/SwiftLint/blob/main/README_CN.md)
+> [한국어](https://github.com/realm/SwiftLint/blob/main/README_KR.md)
+
+## Video Introduction
+
+To get a high-level overview of SwiftLint, we encourage you to watch this
+presentation recorded January 9th, 2017 by JP Simard (transcript provided):
+
+[![Presentation](https://raw.githubusercontent.com/realm/SwiftLint/main/assets/presentation.svg)](https://youtu.be/9Z1nTMTejqU)
 
 ## Installation
 
-### Using [Swift Package Manager](https://github.com/apple/swift-package-manager):
+### Swift Package Manager
 
-> Replace `<version>` with the desired minimum version.
+https://github.com/apple/swift-package-manager
+
+SwiftLint can be used as a [command plugin](#swift-package-command-plugin)
+or a [build tool plugin](#swift-package-build-tool-plugins).
+
+Add the following to your `Package.swift` file:
 
 ```swift
 .package(url: "https://github.com/realm/SwiftLint.git", from: "<version>")
 ```
 
-SwiftLint can be used as a [command plugin](#swift-package-command-plugin) or a [build tool plugin](#swift-package-build-tool-plugins).
+Or to pin to an exact version:
+
+```swift
+.package(url: "https://github.com/realm/SwiftLint.git", exact: "<version>")
+```
+
+> [!IMPORTANT]
+> Replace `<version>` with the desired minimum or exact version.
+
+### Xcode Package Dependency
```

Comment:
Isn't this only useful to develop something against SwiftLint, like an own version with own rules? That's very advanced so probably shouldn't be mentioned here.

Reply:
This is **_not_** for developing something against SwiftLint.

The existence of SwiftLint as a dependency in the package provides access to the Command Plugin and any targets to be linted add SwiftLint as a **_plugin_**, not a target dependency (which is what would be done if developing something against SwiftLint).

Reply:
This is **_not_** for developing something against SwiftLint.

The existence of SwiftLint as a package dependency provides access to the Command Plugin and any targets to be linted add SwiftLint as a **_plugin_**, not a target dependency (which is what would be done if developing something against SwiftLint).

Reply:
"Xcode Package Dependency" sounds like more than adding a plugin. That's what confused me. But that's actually the correct name I think. We should be very clear here, though, that SwiftLint must not be added as a target dependency. Again and again people report issues because of ignoring that. "It is not required" in the box below sounds to lax in that regard.